ui/app/components/clients/current.js

```diff
@@ -0,0 +1,31 @@
+import Component from '@glimmer/component';
+
+export default class Current extends Component {
+  chartLegend = [
+    { key: 'entity_clients', label: 'entity clients' },
+    { key: 'non_entity_clients', label: 'non-entity clients' },
+  ];
+
+  // data for horizontal bar chart in attribution component
+  get topTenNamespaces() {
+    if (!this.args.model.monthly || !this.args.model.monthly.byNamespace) {
+      return null;
+    }
+    return this.args.model.monthly.byNamespace;
```

Comment:
I _think_ you can ditch the whole if statement block if you do a ternary operator like `return this.args.model.monthly?.byNamespace`

Reply:
Awesome! I'll play with that - would definitely like to clean it up. I pulled this over from 1.9 client count work